content/docs/reference/pkg/nodejs/pulumi/alicloud/_index.md

```diff
@@ -0,0 +1,875 @@
+---
+title: "Package @pulumi/alicloud"
+title_tag: "Package @pulumi/alicloud | Node.js SDK"
+linktitle: "@pulumi/alicloud"
+meta_desc: "Explore members of the @pulumi/alicloud package."
+git_sha: "9546b29284c3141a4c805ab6282e4f27a9e31e90"
+---
+
+<!-- WARNING: this page was generated by a tool. Do not edit it by hand. -->
+<!-- To change it, please see https://github.com/pulumi/docs/tree/master/tools/tscdocgen. -->
+
+
+> This provider is a derived work of the [Terraform Provider](https://github.com/terraform-providers/terraform-provider-alicloud)
```

Comment:
Just something to stick in the back of your head, might make sense to try to write a little README, no matter how small, anytime we bring up a new provider. Even if just a link to the docs for configuration, etc.